packages/plugin-inline/InlineNode.ts

```diff
@@ -0,0 +1,13 @@
+import { VNode } from '../core/src/VNodes/VNode';
+import { Format } from './Format';
+
+export class InlineNode extends VNode {
+    formats: Format[] = [];
+    get name(): string {
+        let name = this.constructor.name;
+        for (const format of this.formats) {
+            name += '.' + format.name;
```

Comment:
Wait, don't you have the same logic there ?

https://github.com/odoo-dev/jabberwock/pull/99/commits/e57e8ef3ce4a178b58a890d18500cb3010dc48b9#diff-00e933b8cab4fdbb08b9bc11caf03ceeR10

Isn't that going to end up appending the format names twice ?